banding_removal/fastmri/data/mri_data.py

```diff
@@ -156,21 +157,17 @@ def __init__(self, root, transform, args, fnames=None):
 
             # Compute the size of zero padding in k-space
             # We really should have stored this as an attribute in the hdf5 file
-            try:
-                import ismrmrd
-                hdr = ismrmrd.xsd.CreateFromDocument(
-                    data['ismrmrd_header'][()])
-                enc = hdr.encoding[0]
-                enc_size = (enc.encodedSpace.matrixSize.x,
-                            enc.encodedSpace.matrixSize.y,
-                            enc.encodedSpace.matrixSize.z)
-                enc_limits_center = enc.encodingLimits.kspace_encoding_step_1.center
-                enc_limits_max = enc.encodingLimits.kspace_encoding_step_1.maximum + 1
-                padding_left = enc_size[1] // 2 - enc_limits_center
-                padding_right = padding_left + enc_limits_max
-            except:
-                padding_left = 0
-                padding_right = 0
+            hdr = ismrmrd.xsd.CreateFromDocument(
+                data['ismrmrd_header'][()])
+            enc = hdr.encoding[0]
+            enc_size = (enc.encodedSpace.matrixSize.x,
+                        enc.encodedSpace.matrixSize.y,
+                        enc.encodedSpace.matrixSize.z)
+            enc_limits_center = enc.encodingLimits.kspace_encoding_step_1.center
+            enc_limits_max = enc.encodingLimits.kspace_encoding_step_1.maximum + 1
+            padding_left = enc_size[1] // 2 - enc_limits_center
+            padding_right = padding_left + enc_limits_max
```

Comment:
See usage of [`et_query`](https://github.com/facebookresearch/fastMRI/blob/82e758ac80f2a9b21ebd0609437f0128f75b90cd/fastmri/data/mri_data.py#L25-L57) if you'd like another way to use this. `ismrmrd` has a dependency on an unmaintained XML library so I did some work to remove it.